traffic_portal/app/src/common/modules/form/deliveryService/form.deliveryService.HTTP.tpl.html

```diff
@@ -142,6 +142,16 @@
                 </div>
             </div>
 
+            <div class="form-group" ng-class="{'has-error': hasError(deliveryServiceForm.deepCachingType), 'has-feedback': hasError(deliveryServiceForm.deepCachingType)}">
```

Comment:
deep caching type only pertain to HTTP ds's?

Reply:
Yes, only HTTP DSes have access to the client's actual IP, which we need to provide the proper localization/routing for deep caching. In the future if we can get better localization from DNS DSes using the client subnet extension we might be able to do deep caching for those.

Reply:
I might've spoken too soon! It should pertain to `STEERING` type DSes as well; I'll add that too.